frontend/src/modules/scribble/components/CanvasPanel.vue

```diff
@@ -10,7 +10,7 @@ import Canvas from "./Canvas.vue";
 import BrushSelector from "./BrushSelector.vue";
 import { createBrushStyle } from "../utility/BrushStyleUtils";
 import { createBrushStrokeMessage } from "../utility/WebSocketMessageUtils";
-import { EventBus } from "../eventBus.js";
+import { EventBus } from "@/eventBus.js";
```

Comment:
There a reason we are only doing this for some of the imports?

Reply:
Good point, should find a consistent format for these